sources/packages/web/src/components/common/students/StudentFileUploads.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <tab-container>
```

Comment:
So this one being a component, we must remove the tab container from here and I see already the view/*.vue files using this are having the tab container in it.